### ISSUE TITLE
Low constrast of some colors (WCAG)

### ISSUE THREAD

**Issue by black23** [jajm/koha-staff-interface-redesign#10]
For the sake of good readability and high accessibility of the new interface, we should fine-tune the colour contrasts. Ideally, to comply with WCAG requirements. The contrast ratio should be at least 4.5:1. In some places we are now reaching 2:1, which makes it less readable (for example, smaller green text on a white or grey background).
![obrazek](https://user-images.githubusercontent.com/5516936/182559911-c60c0f04-b107-44c0-8527-0d7d3280e95d.png)

**Comment by black23** [comment]
Examples of possible colors (contrast ration 4.5:1 or better):
- green text on white/grey background #378200
- white text on green background #3C8627

**Comment by mrenvoize** [comment]
Once we've settled on colours, can we get them published in an official style guideline doc somewhere ;)

**Comment by black23** [comment]
Yes! I agree.

Michal

po 5. 9. 2022 v 11:33 odesílatel Martin Renvoize ***@***.***>
napsal:

**Comment by black23** [comment]
From W3 [ about contrast](https://www.w3.org/TR/WCAG21/#contrast-minimum)

**Comment by jajm** [comment]
I think most contrast issues have been solved by the latest patches. Some remain, but master has contrast issues too. Unless I missed something critical, I consider it is outside the scope of the redesign

**Comment by mrenvoize** [comment]
I agree.. we're good here for now.

I see 5 outstanding contrast issues on the main page, but all are trivially solvable as their own bug follow-ups and would be better tracked outside of this patchset in my opinion.. lets concentrate on getting this in and fixing further issues as their own thing.

Example, the separators one is a simple fix on .separator color

```
.separator {
  color: #bfbfbf;
  padding: 0 .2em;
}
```

**Comment by mrenvoize** [comment]
Perfect sort of thing for Academy students ;P

**Comment by mrenvoize** [comment]
Moved to [Bug 31760](https://bugs.koha-community.org/bugzilla3/show_bug.cgi?id=31760) - Low constrast of some colors (WCAG)
